### PR TITLE
Vjsikha 157 cicd challenge question

### PR DESCRIPTION
**Pull request by oak2278**
*Issue #, if available:*

*Description of changes:*
moved challenge question. fixed typo in rolename across multiple pages. typos

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: layouts/shortcodes/cf-launch.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <a
-  href="https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName={{ .Get 1 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 0 }}"
+href="https://console.aws.amazon.com/cloudformation/home?#/stacks/create/review?stackName={{ .Get 0 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 1 }}"
```
Comment (rnzsgh) [inline layouts/shortcodes/cf-launch.html]: Please revert this back.

**Comment by oak2278** [comment]
@brentley Looks like this has some old changes in it. How do i remove the healthcheck and cf-launch commits

File: content/codepipeline/codepipeline.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Click the **Launch** button to create the CloudFormation stack in the AWS Manage
 
 | Launch template |  |  |
 | ------ |:------:|:--------:|
-| CodePipeline & EKS |  {{% cf-launch "ci-cd-codepipeline.cfn.yml" "eksws-codepipeline" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
+| CodePipeline & EKS |  {{% cf-launch "eksws-codepipeline" "ci-cd-codepipeline.cfn.yml" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
```
Comment (rnzsgh) [inline content/codepipeline/codepipeline.md]: Why are you changing this?

File: content/codepipeline/change.md [hunk 3]
```diff
@@ -22,21 +23,11 @@ You should leave the master branch selected.
...
-
-Once the service is built and delivered, run the following command to get the Elastic Load Balancer (ELB) endpoint and open it in a browser.
-If the message is not updated immediately, give Kubernetes some time to deploy the change.
-
-```
-kubectl get services hello-k8s -o wide
-```
-
-{{% notice info %}}
-The EXTERNAL-IP column contains a value that ends with "elb.amazonaws.com" - the full value is the DNS record.
```
Comment (rnzsgh) [inline content/codepipeline/change.md]: Why are you removing this section?

**Comment by vjsikha** [comment]
@oak2278 @brentley I will revert the changes and confirm

File: content/codepipeline/codepipeline.md (hunk 2 above)
Comment (oak2278) [reply content/codepipeline/codepipeline.md]: those should not have been changed by me. I may have had an old version when I started

File: content/codepipeline/change.md (hunk 3 above)
Comment (rnzsgh) [reply content/codepipeline/change.md]: Nm. I cut off the bottom with my comment.

**Comment by rnzsgh** [comment]
LGTM

**Pull request merged**